docs/model_server_rest_api.md

```diff
@@ -180,4 +183,103 @@ A request in [column format](https://www.tensorflow.org/tfx/serving/api_rest#spe
   "outputs": <value>|<(nested)list>|<object>
 }
 ```
-Read more about *Predict API* usage [here](./../example_client/README.md#predict-api-1)
\ No newline at end of file
+Read more about *Predict API* usage [here](./../example_client/README.md#predict-api-1)
+
+## Config Reload API <a name="config-reload"></a>
+* Description
+
+Sends requests via RESTful API to trigger config reloading and get models and DAGs statuses as a response.
+Flow after receiving request:
+1) If config file was changed - reload config. (If reload was needed return code is 201, otherwise 200)
+2) If any model version directory was changed or new version directory was added - reload this model.
+3) If any model that is part of a pipeline was changed or new version dir was added - reload this pipeline.
+4) Trigger Get Model Status API for all models and pipelines, aggregate all responses into one and return it with proper code.
```

Comment:
Hmm, I'm not sure if we use the word "pipelines" externally. Are you sure using DAGs and pipelines interchangeably is okay?

Reply:
instead of trigger Get Model Status API add: In case there are no errors in the reload operation, the response includes the status of all models.